src/lumigo_tracer/libs/__init__.py

```diff
@@ -1 +1,11 @@
 # TODO - do we need to add to our license something that thanks those repositories?
```

Comment:
remove the `TODO`
Replace "customers" with "users"

Reply:
I am waiting for an answer from David to see if we have any limitations / requirements because we are copying source code from external libraries